app/views/vatDetails/details.scala.html

```diff
@@ -18,56 +18,67 @@
 
 @views.html.main_template(title = messages("vatDetails.title"), bodyClasses = None, appConfig = appConfig) {
 
-  @navigationBreadcrumb(
-    links = Map(appConfig.btaHomeUrl -> messages("breadcrumbs.bta")),
-    currentPage = messages("vatDetails.title")
-  )
+    @navigationBreadcrumb(
+        links = Map(appConfig.btaHomeUrl -> messages("breadcrumbs.bta")),
+        currentPage = messages("vatDetails.title")
+    )
 
-  <div class="grid-row form-group">
-    <div class="column-two-thirds">
-      <header class="page-heading push--top"
-              @if(!detailsViewModel.entityName.isDefined){data-metrics="error:hidden-text:vat-overview-entity-name"}>
-        <h1 class="heading-xlarge">@messages("vatDetails.title")</h1>
-        @detailsViewModel.entityName.map { entityName =>
-          <p class="heading-secondary">@entityName</p>
-        }
-        <span class="form-hint">@messages("vatDetails.vrn", user.vrn)</span>
-      </header>
+    <div class="grid-row form-group">
+        <div class="column-two-thirds">
+            <header class="page-heading push--top"
+                @if(!detailsViewModel.entityName.isDefined){data-metrics="error:hidden-text:vat-overview-entity-name"}>
+                <h1 class="heading-xlarge">@messages("vatDetails.title")</h1>
+                @detailsViewModel.entityName.map { entityName =>
+                    <p class="heading-secondary">@entityName</p>
+                }
+                <span class="form-hint">@messages("vatDetails.vrn", user.vrn)</span>
+            </header>
+        </div>
     </div>
-  </div>
 
-  <div class="grid-row">
-    <div class="column-two-thirds">
+    <div class="grid-row">
+        <div class="column-two-thirds">
+
+            <div class="grid-row divider--bottom">
+                @views.html.templates.nextPaymentSection(
+                    detailsViewModel.nextPaymentDueDate,
+                    detailsViewModel.paymentOverdue,
+                    detailsViewModel.paymentError
+                )
+                @views.html.templates.nextReturnSection(
+                    detailsViewModel.nextReturnObligationDueDate,
+                    detailsViewModel.returnObligationOverdue,
+                    detailsViewModel.returnObligationError
+                )
+            </div>
 
-      <div class="grid-row divider--bottom">
-        @views.html.templates.nextPaymentSection(
-          detailsViewModel.nextPaymentDueDate,
-          detailsViewModel.paymentOverdue,
-          detailsViewModel.paymentError
-        )
-        @views.html.templates.nextReturnSection(
-          detailsViewModel.nextReturnObligationDueDate,
-          detailsViewModel.returnObligationOverdue,
-          detailsViewModel.returnObligationError
-        )
-      </div>
+            <div class="grid-row">
+                @if(paymentHistoryEnabled) {
```

Comment:
Instead of using the bool passed in from the controller you can use the appConfig and then overwrite it in the tests to test it appearing in the view